src/registrar/tests/test_admin_domain.py

```diff
@@ -167,12 +167,6 @@ def test_analyst_can_see_inline_domain_information_in_domain_change_form(self):
         expected_organization_name = "MonkeySeeMonkeyDo"
         self.assertContains(response, expected_organization_name)
 
-        # clean up this test's data
-        domain.delete()
-        domain_information.delete()
-        _domain_request.delete()
-        _creator.delete()
-
```

Comment:
They were redundant. And domain.delete() was failing because Host was not deleted first. In this case, _creator is not deleted properly I think. But I think it is okay.